package.json

```
@@ -21,11 +21,11 @@
     "index.js"
   ],
   "scripts": {
-    "setup": "yarn install && yarn allow-scripts",
+    "setup": "yarn install",
     "test": "jest",
     "test:watch": "jest --watch",
     "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore",
```

The module template currently has this, so not sure if you want to make this change as well:

```suggestion
    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
```